.gitignore

```diff
@@ -43,3 +43,5 @@ notes*
 /build
 
 .DS_Store
+node_modules
+package-lock.json
```

Comment:
I've opted to ignore `package-lock.json` (for now).  The only thing in there is prettier, I have the version explicitly set in `package.json`, and I don't really want it locking the node version right now.

(Even though I did lock nodejs to latest LTS in `.tool-versions`).